stanford-earth/SE3_BLT
# EARTH-96: Enable new paragraph modules.

# READY FOR REVIEW

# Summary
- Turns on the separated out paragraph features.

# Needed By (Date)
- 2017.06.14

# Urgency
- Moderate

# Steps to Test

0. Refresh your local environment (optional)
1. Check out this branch
2. Checkout branch EARTH-96 of the stanford_paragraph_types module if (https://github.com/SU-SWS/stanford_paragraph_types/pull/30) hasn't been merged yet. If merge has been made ensure you update your environment with the latest.
3. Run `drush csim`
4. Run `drush features-import-all`
5. Clear caches
6. Navigate to `/node/add/complex_page`
7. Ensure all of the existing paragraph types are available
8. Add and create a couple of them, then save the node.
9. Review that the paragraph types work.

# Affected Projects or Products
- https://github.com/SU-SWS/stanford_paragraph_types/pull/30
- https://github.com/SU-SWS/stanford_paragraph_types/pull/29

# Associated Issues and/or People
- EARTH-91
- EARTH-96

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
